README.md

```diff
@@ -60,6 +60,7 @@ To prepare your system to deploy Clearwater manually, run:
 To start the Clearwater services, run:
 
     sudo docker network create --driver bridge clearwater_nw
+    sudo docker run -d --net=clearwater_nw --name etcd quay.io/coreos/etcd:v2.2.5 -name etcd0 -advertise-client-urls http://etcd:2379,http://etcd:4001 -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 -initial-advertise-peer-urls http://etcd:2380 -listen-peer-urls http://0.0.0.0:2380  -initial-cluster etcd0=http://etcd:2380 -initial-cluster-state new
```

Comment:
A bit of a mouthful!  Should we at least break it onto multiple lines (with `\`s) so that we can read it?  In fact, should we break this out as a separate step?  (One of the arguments for running this separately is that it might be a third-party etcd, so it would be good to make that clear.)